version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.11.0-SNAPSHOT"
+version in ThisBuild := "0.11.0-SNAPSHOT-7868cf219bebd493ce2ae5b7b51a3f85c9f7770e"
```

Comment:
This is not intentional I suppose. We should make sure to revert this before merging.